Gems/Multiplayer/Code/Source/Components/NetBindComponent.cpp

```diff
@@ -177,7 +179,9 @@ namespace Multiplayer
             // The component hasn't been pre-setup with NetworkEntityManager yet. Setup now.
             const AZ::Name netSpawnableName = AZ::Interface<INetworkSpawnableLibrary>::Get()->GetSpawnableNameFromAssetId(m_prefabAssetId);
 
-            AZ_Assert(!netSpawnableName.IsEmpty(),
+            // In client-server the level asset is a temporary Root.network.spawnable and isn't registered in time.
+            // But this is not needed in client-server mode.
```

Comment:
```suggestion
            // In client-server the level asset is a temporary Root.network.spawnable and is not expected to be registered in time.
```

This isn't reading clearly to me. Maybe this suggestion is what you're going for?

Reply:
Yeah, that is good.